website/static/js/osfHelpers.js

```diff
@@ -38,18 +39,12 @@ var growl = function(title, message, type) {
 var apiV2Url = function (pathString, paramsObject, apiPrefix){
```

Comment:
I really like this suggestion; it's cleaner to use and far more idiomatic. Thanks!

I've gone ahead and pushed this change. Changing the call signature will affect anyone who has already started using the original version- but it's only been a day, so @brianjgeiger et al shouldn't be terribly inconvenienced.